oc_anonymizer.py

```diff
@@ -17,47 +17,59 @@ class PlistStripper:
     def __init__(self):
         self.plist = load_plist()
         print(self.plist['PlatformInfo']['Generic']['ROM'])
-        self.censor_misc_boot()
-        self.censor_misc_debug()
-        self.censor_misc_security()
-        self.censor_platforminfo_generic()
-        self.censor_uefi_apfs()
+        self.reset_misc_boot()
+        self.delete_misc_blessoverride()
+        self.reset_misc_debug()
+        self.delete_misc_entries()
+        self.reset_misc_security()
+        self.delete_platforminfo_generic()
+        self.disable_uefi_apfs()
         self.dump()
 
-    def censor_misc_boot(self) -> None:
+    def reset_misc_boot(self) -> None:
         """ Sets Misc/Boot/LauncherOption to Disabled to avoid registering the launcher option in the firmware
         preferences for persistence"""
 
         self.plist['Misc']['Boot']['LauncherOption'] = 'Disabled'
 
-    def censor_misc_debug(self) -> None:
+    def delete_misc_blessoverride(self) -> None:
+        """ Deletes custom BlessOverride entries from config.plist"""
+
+        self.plist['Misc']['BlessOverride'] = []
+
+    def reset_misc_debug(self) -> None:
         """ Sets Misc/Debug/Target to 3"""
 
         self.plist['Misc']['Debug']['Target'] = 3
 
-    def censor_misc_security(self) -> None:
+    def delete_misc_entries(self) -> None:
+        """ Deletes custom bootloader entries from config.plist"""
+
+        self.plist['Misc']['Entries'] = []
+
+    def reset_misc_security(self) -> None:
         """Sets
             -> Misc/Security/ApECID to 0 to disallow using personalised Apple Secure Boot identifiers (unsupported from
                 Monterey and later)
             -> Misc/Security/ScanPolicy = 0 to allow OpenCore's operating system detection policy
-            -> Misc/Security/SecureBootModel = Default to configure Apple Secure Boot hardware model
+            -> Misc/Security/SecureBootModel = Disabled. Disables Apple Secure Boot hardware model to avoid issues during Installation. Re-enable in Post-Install so System Updates work when using an SMBIOS of a Mac model with a T2 Security Chip.
             -> Misc/Security/Vault = Optional to disable OpenCore's vaulting mechanism
         """
 
         self.plist['Misc']['Security']['ApECID'] = 0
         self.plist['Misc']['Security']['ScanPolicy'] = 0
-        self.plist['Misc']['Security']['SecureBootModel'] = 'Default'
+        self.plist['Misc']['Security']['SecureBootModel'] = 'Disabled'
```

Comment:
I'd rather set it to `Default` so users won't worry about fetching new updates. But up to your choice

Reply:
From my experience having SecureBootModel enabled is the number one reason users complain about EFIs not working. The Readme suggests to enable it in Post-Install.